setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='scrapy-html-storage',
-    version='0.2.0',
+    version='0.2.1',
```

Comment:
This library is versioned according to http://semver.org/.
The last number should be incresed only for hotfixes.
But some of these commits introduce breaking changes, e.g. https://github.com/povilasb/scrapy-html-storage/pull/2/commits/0dd89ff19e00c56487401757557483e6b90446d6

I would go with `0.3.0` version.